index.html

```diff
@@ -4479,6 +4479,50 @@ <h2>HTTP</h2>
         <section class="informative">
           <h2>Credential Type-Specific Processing</h2>
 
+          <p>
+RDF [[?RDF11-CONCEPTS]] is a technology for modeling graphs of statements. Each statement is a
+single <i>subject&#x2192;property&#x2192;value</i> (also known as
+<i>entity&#x2192;attribute&#x2192;value</i>) relationship, which is referred to
+as a <a>claim</a> in this specification. JSON-LD [[?JSON-LD11]] is a technology that enables
+the expression of RDF using idiomatic JSON, enabling developers familiar with
+JSON to write applications that consume RDF as JSON. In general, subjects are
+expressed as JSON objects with each property and value of the subject as a JSON
+key and value, respectively. A special affordance is made to express an
+identifier of a subject, if necessary, using the <code>id</code> key in this
+specification. See
+<a data-cite="?JSON-LD11#relationship-to-rdf">Relationship of JSON-LD to RDF</a>
+for more details.
```

Comment:
Doesn't some of this repeat stuff already in https://w3c.github.io/vc-data-model/#claims?

Why are we saying that a "special affordance" is made to express an identifier of a subject? We say that here, right? https://w3c.github.io/vc-data-model/#syntactic-sugar

Reply:
> Why are we saying that a "special affordance" is made to express an identifier of a subject?

Because the preceding sentence might imply to some readers that `id` is a property of a subject rather than being special in that it holds the identifier for the subject instead. We can strike the sentence if we think that readers won't get confused over that.